### DISCUSSION TITLE
AutoComplete Delaying results

### DISCUSSION THREAD

**Discussion by andrewtquick** [rsuite/rsuite#3123] [Q&A]
I have an AutoComplete component in my app, and my data for the component is just under 5,000 different options. When searching, the component freezes as it searches for the value.

Is there a way to implement a delay of showing the results? Allow the user to enter some text, wait a second or two for them to stop typing then filter and show results?

**Comment by SevenOutman** [comment]
Hi @andrewtquick, there's currently no such API for AutoComplete. But I think it's good to add support. Thanks for raising this use case!
